### DISCUSSION TITLE
Network Error: JSON.parse: unexpected non-whitespace character

### DISCUSSION THREAD

**Discussion by mygrexit** [requarks/wiki#3244] [Error / Bug Report]
Hello Guys,

i am using the latest wiki.js docker image with traefik as a reverse proxy.

I am receiving the following error when im trying to create a page:

`Network Error: JSON.parse: unexpected non-whitespace character after JSON data at line 1 column 5 of the JSON data`

My feeling is that this is reverse proxy related since i havent had this issue without it.

What can i do to debug it?

**Comment by mygrexit** [comment]
I can confirm that this error only appears when using a reverse proxy.

![grafik](https://user-images.githubusercontent.com/33792951/111837467-3822be00-88f8-11eb-8149-e3387d450b21.png)

**Comment by mygrexit** [comment]
Digged through some old issues and found out that this is due to Traefiks basic auth that i had activated.
I think that this should be added to the docs. Maybe it already is, but i didnt find it.

**Reply by GabrielLins64** [reply]
Hi, could you please tell me how did you solve it with Traefik auth? I'm having the exact same issue. Thanks!

**Reply by mygrexit** [reply]
Sorry, but I can't remember and I am using a different proxy nowadays.

**Reply by GabrielLins64** [reply]
Oh, ok. Thanks anyway 🥲
